fluentdhec/pyhec.py

```diff
@@ -1,24 +1,18 @@
 import requests
 import os
+from typing import Tuple
 
 
-class PyHEC:
-    def __init__(self, token, host, port='443'):
-        self.token = token
-        self.uri = f"https://{host}:{port}/services/collector"
-
-    def send(self, payload):
-        headers = {'Authorization': f'Splunk {self.token}'}
-        timeout = int(os.getenv('SPLUNK_HEC_TIMEOUT', '10'))
-        try:
-            r = requests.post(
-                self.uri,
-                payload,
-                headers=headers,
-                verify=True,
-                timeout=timeout,
-            )
-            return r.status_code, r.text,
-        except requests.exceptions.Timeout:
-            print(f"ERROR: PyHEC:send: Exceeded timeout: {timeout}")
-            return False, False
+def send(token: str, host: str, payload: str, port: str = "443") -> Tuple[int, str]:
+    """
+    Sends a request to a Splunk HEC.
+    """
+    uri = f"https://{host}:{port}/services/collector"
+    headers = {"Authorization": f"Splunk {token}"}
+    timeout = int(os.getenv("SPLUNK_HEC_TIMEOUT", "10"))
+    try:
+        r = requests.post(uri, payload, headers=headers, verify=True, timeout=timeout,)
```

Comment:
@denizgenc the trailing comma here on timeout indicated to me that you're not auto-formatting with black.
Don't worry about fixing this, my next commit will.

Install black globally (so atom can find it) and use https://atom.io/packages/python-black :ok_hand: